cmd/filegen/main.go

```diff
@@ -275,44 +271,24 @@ func generateFiles(ctx *cli.Context) error {
 	}
 
 	var (
-		err                        error
-		initialBalancesSkFile      *os.File
-		initialBalancesSkPlainFile *os.File
-		initialBalancesPkPlainFile *os.File
-		initialNodesSkFile         *os.File
-		initialNodesSkPlainFile    *os.File
-		initialNodesPkPlainFile    *os.File
-		genesisFile                *os.File
-		nodesFile                  *os.File
-		txgenAccountsFile          *os.File
-		pkHex                      string
-		skHex                      []byte
-		suite                      crypto.Suite
-		balancesKeyGenerator       crypto.KeyGenerator
+		err                  error
+		walletKeyFile        *os.File
+		validatorKeyFile     *os.File
+		genesisFile          *os.File
+		nodesFile            *os.File
+		txgenAccountsFile    *os.File
+		pkHex                string
```

Comment:
is this still pkHex if the txSignKeyFormat is bech32? Should it be name just pk?

Reply:
discussed